test/fixtures/export/Do not caps-protect name fields #384.bibtex

```diff
@@ -34,12 +34,12 @@
 }
 
 @book{bovendeert,
-  author = {{boven d'\relax{} Eert}, Christianus},
+  author = {{boven d'Eert}, Christianus},
```

Comment:
Only DPs do. Again, given the limitations of bibtex, the best that can be done (IMHO) is (1) to let bibtex parse DPs as prefixes (the space is needed for *parsing*, plus `\relax` to later remove this space betw. punctuation and Last) – and (2) let NDPs become part of a (multi-part) Last ({van Gogh}), but, optionally, sort by the original Last (Gogh).